examples/kind/tests/sample_test.go

```diff
@@ -35,6 +78,6 @@ func TestFail(t *testing.T) {
 }
 
 func TestTimeout(t *testing.T) {
-	logrus.Infof("test timeout for 5 seconds:" + os.Getenv("KUBECONFIG"))
-	<-time.After(5 * time.Second)
+	logrus.Infof("test timeout for 10 seconds:" + os.Getenv("KUBECONFIG"))
+	<-time.After(20 * time.Second)
```

Comment:
Why this is increased?

Reply:
Test timeout 5 -> 10 seconds, because `TestSuite` is running for about 5 seconds and so 5 seconds timeout is not enough.
Waiting 5 -> 20 seconds, because with 10 seconds waiting it doesn't always fail with timeout.